src/grt/src/MakeWireParasitics.cpp

```diff
@@ -201,26 +220,31 @@ void MakeWireParasitics::makeParasiticsToPin(Pin& pin)
                GRT,
                "est_rc",
                1,
-               "{} -> {} ({:.2f}, {:.2f}) {:.2f}u layer={} r={} c={}",
+               "{} -> {} ({:.2f}, {:.2f}) {:.2f}u layer={} r={} via_res={} c={}",
```

Comment:
Are we still using the super-fragile parsing script for RC correlation? This might break it FYI. I think there was agreement to write/read JSON but no one has had time to implement it.

Reply:
No I think we use write_net_rc.tcl which uses sta's pi model